bleak/__init__.py

```diff
@@ -402,7 +423,10 @@ async def disconnect(self) -> bool:
 
     async def pair(self, *args, **kwargs) -> bool:
         """
-        Pair with the peripheral.
+        Pair with the specified GATT server.
+
+        This method may not be available (or needed) for some backends.
```

Comment:
Let's be explicit about which backends don't support it. IIRC, only macOS doesn't support this and it will instead pair the first time a characteristic that requires authentication is accessed.

Reply:
```suggestion
        This method is not available on macOS. Instead of manually initiating
        paring, the user will be prompted to pair the device the first time
        that a characteristic that requires authentication is read or written.
```